--- a/src/Makefile.am.orig2	2019-09-05 03:43:14.922400000 +0200
+++ b/src/Makefile.am	2019-09-05 04:00:04.513484000 +0200
@@ -43,6 +43,8 @@
 bin_PROGRAMS = rtorrent
 
 rtorrent_LDADD = \
+	../gnulib/unistd.o \
+	../gnulib/random_r.o \
 	libsub_root.a \
 	ui/libsub_ui.a \
 	core/libsub_core.a \
@@ -55,4 +57,4 @@
 rtorrent_SOURCES = \
 	main.cc
 
-AM_CPPFLAGS = -I$(srcdir) -I$(top_srcdir)
+AM_CPPFLAGS = -I$(srcdir) -I$(top_srcdir) -I$(top_srcdir)/gnulib
